src/main/java/com/alkemy/somosmas/mappers/OrganizationMapper.java

```diff
@@ -0,0 +1,44 @@
+package com.alkemy.somosmas.mappers;
+
+import com.alkemy.somosmas.dtos.OrganizationBasicDTO;
+import com.alkemy.somosmas.models.Organization;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class OrganizationMapper {
+
+
+
+    public OrganizationBasicDTO organizationModel2BasicDto(Organization organization){
+
+        OrganizationBasicDTO dto= new OrganizationBasicDTO();
+
+        dto.setName(organization.getName());
+        dto.setImage(organization.getImage());
+        dto.setPhone(organization.getPhone());
+        dto.setAddress(organization.getAddress());
+
+        return dto;
+    }
+
+    public List<OrganizationBasicDTO> organizationModelList2BasicDtoList(List<Organization> organizationList){
+
+        List<OrganizationBasicDTO> dtosList= new ArrayList<>();
+
+        for(Organization model : organizationList){
```

Comment:
![0_kLriPzotQNnsh2Cj](https://user-images.githubusercontent.com/14932921/167704966-4e55f60f-4abe-45ec-b65d-fecbd428bdcd.png)

Usemos streams para recorrer la listas concurrentemente y hacer el map al DTO
https://medium.com/javarevisited/how-to-use-streams-map-filter-and-collect-methods-in-java-1e13609a318b